app/javascript/components/layout/CustomTable.js

```diff
@@ -197,7 +197,7 @@ class CustomTable extends React.Component {
               })}
               {!this.props.rowData?.length && (
                 <tr>
-                  <td colSpan={this.props.columnData?.length} className="text-center">
+                  <td colSpan={this.props.columnData?.length + (this.props.isEditable ? 0 : 1) + (this.props.isSelectable ? 0 : 1)} className="text-center">
```

Comment:
out of curiosity, was this always a ui bug that we just didn't notice before?

Reply:
I guess? I couldn't find a JIRA for it when I discovered it and it was easy enough to fix

Reply:
Great catch!